platform/emdrv/nvm3/src/ccp_flash_prg.c

```diff
@@ -0,0 +1,97 @@
+/*******************************************************************************
+ * @file  FlashPrg.c
+ * @brief
+ *******************************************************************************
+ * # License
+ * <b>Copyright 2020 Silicon Laboratories Inc. www.silabs.com</b>
+ *******************************************************************************
+ *
+ * The licensor of this software is Silicon Laboratories Inc. Your use of this
+ * software is governed by the terms of Silicon Labs Master Software License
+ * Agreement (MSLA) available at
+ * www.silabs.com/about-us/legal/master-software-license-agreement. This
+ * software is distributed to you in Source Code format and is governed by the
+ * sections of the MSLA applicable to Source Code.
+ *
+ ******************************************************************************/
+
+#include "ccp_flash_prg.h" // FlashOS Structures
+#include "RS1xxxx_9117.h"
+#include "ccp_flash_intf.h"
+#include <stdint.h>
+
+/*
+ *  Initialize Flash Programming Functions
+ *    Parameter:      adr:  Device Base Address
+ *                    clk:  Clock Frequency (Hz)
+ *                    fnc:  Function Code (1 - Erase, 2 - Program, 3 - Verify)
+ *    Return Value:   0 - OK,  1 - Failed
+ */
+int Init(unsigned long adr, unsigned long clk, unsigned long fnc) {
```

Comment:
For all those prototypes you used an `int` to return 0 or 1. using a Bool would use 1 byte instead of 4.
Also, I suggest using a return code/enum with a relevant name instead of directly using 0 and 1